src/netappfiles-preview/HISTORY.rst

```diff
@@ -2,6 +2,18 @@
 
 Release History
 ===============
+1.0.0b2
```

Comment:
It seems there are no release notes for `1.0.0b1`.

Reply:
You mean thye previous version? Do we need to add notes for that  here?

Reply:
Please add the history for 1.0.0b1 in later PRs